micro_dl/utils/mp_utils.py

```diff
@@ -141,38 +141,14 @@ def create_and_write_mask(
 
     # save masks as additional channel
     position_masks = position_masks.astype(position.data.dtype)
+    new_channel_name = channel_name + '_mask'
     io_utils.add_channel(
```

Comment:
This function is also only called in this file. Maybe we should remove `io_utils.py` altogether since we don't have 'untracked' arrays anymore.